data/cpu-x.desktop.in

```diff
@@ -11,3 +11,4 @@ Icon=@APP_EXEC@
 Type=Application
 Categories=GTK;System;
 Terminal=false
+Keywords=CPU;system;core;speed;clock;rate;overclock;Intel;AMD;motherboard;
```

Comment:
This tool does not allow user to overclock its CPU. Please fix it before I merge, thanks.